src/traits.rs

```diff
@@ -787,6 +800,12 @@ pub trait Monty:
     /// Returns one in this representation.
     fn one(params: Self::Params) -> Self;
 
+    /// Returns the parameter struct used to initialize this object.
+    fn params(&self) -> &Self::Params;
+
+    /// Access the value in Montgomery form.
+    fn as_montgomery(&self) -> &Self::Integer;
```

Comment:
I'm not wild about this name (or some of the others) but we can bikeshed it in another issue

Reply:
It just mimics the inherent method name we already have.

Reply:
Yeah, definitely, I just find the existing naming confusing. I should've prefixed it with "sidebar"